graylog2-server/src/main/java/org/graylog2/bootstrap/CmdLineTool.java

```diff
@@ -388,6 +392,22 @@ public void configure(Binder binder) {
         }
     }
 
+    /**
+     * Set up a separate injector, containing only the configuration bindings. It can be used to look up configuration
+     * values in modules at binding time.
+     */
+    protected Injector setupConfigInjector(NamedConfigParametersModule configModule) {
+        try {
+            return Guice.createInjector(Stage.PRODUCTION, Collections.singletonList(configModule));
```

Comment:
How about enabling explicit bindings for the config injector? That way we make sure that we don't depend on anything unwanted. Or is this not possible with the config bindings?

Reply:
good idea, i'll try that!